#Device Contact
Header header
uint8 state
uint8 fromGSR
uint8 fromHR